spec/components/publish/providers/school_placements/explainer_component_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Publish::Providers::SchoolPlacements::ExplainerComponent, type: :component do
+  it 'renders School Placement information for the Provider' do
+    render_inline(described_class.new)
+
+    expect(page).to have_text('Add the schools you can offer placements in. A placement school is where candidates will go to get classroom experience.', normalize_ws: true)
```

Comment:
What does `normalize_ws: true` do? Not seen that before

Reply:
I normailzes the whitespace. Removes extra spaces and new lines etc so the comparison doesn't fail on an extra space

Reply:
Nice